ui/Modal/Search.svelte

```diff
@@ -0,0 +1,171 @@
+<script lang="typescript">
+  import { push } from "svelte-spa-router";
+  import { createEventDispatcher } from "svelte";
+
+  import * as notification from "../src/notification";
+  import * as path from "../src/path";
+  import type { Project } from "../src/project";
+  import * as remote from "../src/remote";
+  import {
+    projectRequest as request,
+    projectSearch as store,
+    reset,
+    requestProject,
+    searchProject,
+    urnValidationStore,
+  } from "../src/search";
+  import { ValidationStatus } from "../src/validation";
+
+  import { Icon, Input } from "../DesignSystem/Primitive";
+  import { Remote, TrackToggle } from "../DesignSystem/Component";
+
+  let id: string;
```

Comment:
Wait, what's the `id` used for? I see that upon validation you set it as the Urn minus `rad:git`, but I can't see where that's used :thinking:

Reply:
It's shown next to the track button in the search resullt.